src/SourceBuild/content/eng/build.sourcebuild.targets

```diff
@@ -15,45 +15,17 @@
     <MSBuild Projects="$(RepoProjectsDir)$(RootRepo).proj" Targets="WritePrebuiltUsageData;ReportPrebuiltUsage" />
   </Target>
 
-  <Target Name="DiscoverSymbolsTarballs"
```

Comment:
Hmm, we need to extract original `symbols.<repo>` packages. Is this now done in a different target/file?

Reply:
Yes. That's now in repo-projects/Directory.Build.targets.